### PR TITLE
Ruby Style guide updates, add code climate, add code coverage, bump version

### PR DESCRIPTION
**Pull request by usbsnowcrash**

### CODE REVIEW

File: lib/smartling_rails/smartling_processor.rb [hunk 1]
```diff
@@ -2,7 +2,6 @@
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
-#
```
Comment (ausmith) [inline lib/smartling_rails/smartling_processor.rb]: Hmmm, I don't know that we want to delete this line?

File: lib/smartling_rails/smartling_processor.rb [hunk 2]
```diff
@@ -32,24 +29,22 @@ def supported_locales
...
-      SmartlingRails.print_msg "Uploading the english file to smartling to process:", true
-      upload_english_file()
+      SmartlingRails.print_msg('Uploading the english file to smartling to process:', true)
+      upload_english_file
     end
 
-    def upload_english_file()
-      SmartlingRails.print_msg "uploading #{local_file_path_for_locale('en-us')} to #{upload_file_path}"
+    def upload_english_file
+      SmartlingRails.print_msg("uploading #{local_file_path_for_locale('en-us')} to #{upload_file_path}")
```
Comment (ausmith) [inline lib/smartling_rails/smartling_processor.rb]: We prefer spaces in string interpolations. Do you want to add those while touching these files?

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -2,22 +2,27 @@
...
 require 'yaml'
+require 'smartling_rails'
+require 'webmock/rspec'
+WebMock.disable_net_connect!(allow: 'codeclimate.com')
 
+require 'simplecov'
+require 'codeclimate-test-reporter'
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+    SimpleCov::Formatter::HTMLFormatter,
+    CodeClimate::TestReporter::Formatter
```
Comment (ausmith) [inline spec/spec_helper.rb]: Isn't this block indented once too far?

File: lib/smartling_rails/smartling_processor.rb (hunk 2 above)
Comment (usbsnowcrash) [inline lib/smartling_rails/smartling_processor.rb]: Im fine adding some space

File: lib/smartling_rails/smartling_processor.rb (hunk 1 above)
Comment (usbsnowcrash) [inline lib/smartling_rails/smartling_processor.rb]: meh, I dont really care either way

**Pull request merged**